### PR TITLE
feat/avax-support Adding Avax support

### PR DESCRIPTION
**Pull request by fqlx**
- Adding Avax support
- Includes Avax Mainnet and Fuji deployments
- Deterministic deployment is disabled for Avax, the contracts aren't deployed for Avax

### CODE REVIEW

**Review by josojo — APPROVED** [review]
PR looks great.

thanks a lot of the deployment. 
I only double-checked the parameters of the avax deployment, as the testnet is not so important for us.

**Comment by josojo** [comment]
I think, you need to update the yarn.lock file, in order to pass tests

**Comment by fqlx** [comment]
Looks like the lint job is failing in master.  Please review https://github.com/gnosis/ido-contracts/pull/140 and I'll rebase this PR.

File: yarn.lock [hunk 1]
```diff
@@ -8765,11 +9770,11 @@ solhint-plugin-prettier@^0.0.5:
     prettier-linter-helpers "^1.0.0"
 
 solhint@^3.3.2:
-  version "3.3.2"
-  resolved "https://registry.yarnpkg.com/solhint/-/solhint-3.3.2.tgz#ebd7270bb50fd378b427d7a6fc9f2a7fd00216c0"
-  integrity sha512-8tHCkIAk1axLLG6Qu2WIH3GgNABonj9eAWejJbov3o3ujkZQRNHeHU1cC4/Dmjsh3Om7UzFFeADUHu2i7ZJeiw==
+  version "3.3.6"
```
Comment (josojo) [inline yarn.lock]: I think the ci is failing due to these updates

**Comment by josojo** [comment]
I don't think it's failing on main branch... all the hooks are green there. I would pull the yarn.lock again and then it should work, even with your package.json changes.

If you tried this before, let me know. then I also wanna give it a try.

File: yarn.lock (hunk 1 above)
Comment (fqlx) [reply yarn.lock]: Could you try running the pipeline again?

**Comment by josojo** [comment]
Here, I tested it quickly with your package.json: https://github.com/gnosis/ido-contracts/pull/141, seems like its really just the yarn lock

**Comment by josojo** [comment]
Awesome. Thanks

**Pull request merged**